### ISSUE TITLE
Add Typescript support

### ISSUE THREAD

**Issue by kihayu** [tpenaranda/vue-cryptojs#7]
Hi. Would it be possible that you integrate a `types.ts` file to use your dependency in Typescript projects? Especially when Vue 3 brings good Typescript support it would be a good improvement for your project. Thanks

**Comment by jmohler1970** [comment]
I have the same problem

```
14:25 Could not find a declaration file for module 'vue-cryptojs'. '/xxxxxx/node_modules/vue-cryptojs/dist/vue-cryptojs.min.js' implicitly has an 'any' type.
  Try `npm install @types/vue-cryptojs` if it exists or add a new declaration (.d.ts) file containing `declare module 'vue-cryptojs';`
```

**Comment by jmohler1970** [comment]
For anyone that might be stuck, I used

`// @ts-ignore:disable-next-line`

To get around this issue

**Comment by tpenaranda** [comment]
Fixed on 2.4.0 version.
